api_specification/README.md

```diff
@@ -0,0 +1,2 @@
+# Swagger JSON
+This is a swagger JSON built by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.
```

Comment:
Presuming you realise:
We can and should add to this ourselves, replacing it with our own readme